src/services/consortiaService.js

```diff
@@ -32,6 +34,19 @@ export const fetchConsortium = ({ okapi }, tenant) => {
     .then(data => data.consortia[0]);
 };
 
+export const fetchConsortiumMembers = async ({ okapi, user }) => {
+  const consortium = user?.user?.consortium || await fetchConsortium({ okapi }, okapi.tenant);
+
+  return fetch(`${okapi.url}/consortia/${consortium.id}/tenants?limit=${LIMIT_MAX}`, {
+    headers: {
+      [OKAPI_TENANT_HEADER]: okapi.tenant,
+      [CONTENT_TYPE_HEADER]: 'application/json',
+    },
+  })
+    .then(resp => resp.json())
+    .then(data => orderBy(data.tenants || [], 'name'));
+};
+
```

Comment:
I understand that pre-login, this component won't render inside the react query provider in stripes-core, but I wonder if a better implementation would be to do just that, so we could use react-query here, rather than introduce yet another pattern (plain old fetch) for making API calls. What do you think?

Reply:
actually `fetch` and `react-query` are totally different things, react-query requires queryFn that makes a request (under the hood it's always fetch or mb XMLHttpRequest)

in general we reuse this utils in hooks where applicable (e.g. fetchConsortiumMembers and useConsortiumMembers) and do not introduce new pattern

and what we can compare here is ky vs fetch